Assets/MixedRealityToolkit-Examples/Demos/Boundary/Scripts/BoundaryVisualizer.cs

```diff
@@ -0,0 +1,113 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities;
+using Microsoft.MixedReality.Toolkit.Internal.Interfaces;
+using Microsoft.MixedReality.Toolkit.Internal.Managers;
+using UnityEngine;
+using UnityEngine.Experimental.XR;
+
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos
+{
+    /// <summary>
+    /// Demo class to show different ways of using the boundary API.
+    /// </summary>
+    public class BoundaryVisualizer : MonoBehaviour
+    {
+        [SerializeField]
+        [Tooltip("Material used to draw the boundary geometry.")]
+        private Material boundsMaterial = null;
+
+        [SerializeField]
+        [Tooltip("Material used to draw the area outside of the boundary geometry.")]
+        private Material outOfBoundsMaterial = null;
+
+        [SerializeField]
+        [Tooltip("Material used to draw the area inside of the inscribed rectangle.")]
+        private Material inscribedRectangleMaterial = null;
+
+        /// <summary>
+        /// The configured boundary system manager.
+        /// </summary>
+        private IMixedRealityBoundarySystem boundaryManager = null;
+
+        private void Start()
+        {
+            MixedRealityManager mixedRealityManager = MixedRealityManager.Instance;
```

Comment:
This needs to use the standard approach for getting a manager with the private property above, not in start, e.g.

```        private IMixedRealityInputSystem inputSystem = null;
        protected IMixedRealityInputSystem InputSystem => inputSystem ?? (inputSystem = MixedRealityManager.Instance.GetManager<IMixedRealityInputSystem>());
```

Reply:
Yeah if it's an instance there's no reason to cache this reference to the MR Manager.